bool start_sign # true : start false : stop
---
bool result